releasenotes/notes/remove-name-attribute-from-dagcircuit-wires-b54cd6abfce95da4.yaml

```diff
@@ -0,0 +1,6 @@
+---
+other:
```

Comment:
I was thinking this might be an upgrade note instead, but I guess it is kind of an edge case. It's unlikely that people are depending on the `name` field on the edge dict of a `DAGCircuit`

Reply:
Sure, moved in 664485e5dc .